examples/python/learning_stable_baselines3.py

```diff
@@ -86,13 +109,18 @@ def wrap_env(env):
         env_kwargs=dict(frame_skip=FRAME_SKIP),
     )
 
-    agent = PPO("CnnPolicy", envs, n_steps=N_STEPS, verbose=1)
+    agent = PPO("MultiInputPolicy", envs, n_steps=N_STEPS, verbose=2)
 
     # Do the actual learning
     # This will print out the results in the console.
     # If agent gets better, "ep_rew_mean" should increase steadily
+
     try:
-        agent.learn(total_timesteps=TRAINING_TIMESTEPS, progress_bar=True)
+        agent.learn(
+            total_timesteps=TRAINING_TIMESTEPS,
+            progress_bar=True,
+            tb_log_name=f"{args.exp}",
```

Comment:
Isn't it still related to TensorBoard? Actually, maybe you would like to keep it, but this would require a small tutorial on how to set it up for total newbies. Otherwise, let's remove that.